R/plot_plate.R

```diff
@@ -0,0 +1,273 @@
+require(ggplot2)
+require(grid)
+require(png)
+
+
+#' Plot a 96-well plate with colored wells
+#'
```

Comment:
Will we support other types of plates?

Reply:
Right now, we were supposed to focus on 96 well plates as the laboratories we work with don't use 384 well plates. In theory, it is possible to implement it in the same way, but a new background picture is needed, and new sets of parameters that specify the size of wells, title size, etc.

Reply:
Good, we will think about that in the future